teams/3/portfolio-performance.html

```diff
@@ -12,241 +12,382 @@
         <link rel="stylesheet" href="../../css/main.css" />
         <link rel="stylesheet" href="../../css/navigation.css" />
         <link rel="stylesheet" href="../../css/list.css" />
+        <link rel="stylesheet" href="./styles.css" />
     </head>
     <body>
         <header class="flex justify-between">
```

Comment:
please do not use the class as flex, justify, or padding, class needs to reflect the element meaning for the page but not how they are aligned or displayed, I have mentioned this several times. Ans you have it all over. Also the commented images are all failing, the src should be updated for all of them